src/index.html

```diff
@@ -11,7 +11,199 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class = "fieldset">
```

Comment:
do not add spaces between the attribute key and value.
check all places please
```suggestion
      <fieldset class="fieldset">
```